### PR TITLE
#596 fixed null values problem in CommitPropertiesConverter

### PR DESCRIPTION
**Pull request by tau3**

### CODE REVIEW

**Comment by bartoszwalacik** [comment]
thanks, we will check it

File: javers-core/src/test/groovy/org/javers/core/json/typeadapter/commit/CommitPropertiesConverterTest.groovy [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+class CommitPropertiesConverterTest extends Specification {
+    def 'should filter out keys with null values'() {
+        given:
+        final properties = ['name': 'Skif', 'breed': 'scottish straight', 'age': null]
+
+        when:
+        final array = CommitPropertiesConverter.toJson(properties) as Iterable<JsonElement>
+
+        then:
+        final keys = StreamSupport.stream(array.spliterator(), false)
```
Comment (bartoszwalacik) [inline javers-core/src/test/groovy/org/javers/core/json/typeadapter/commit/CommitPropertiesConverterTest.groovy]: did you mean `def keys = array.collect {it['key'].getAsString()}` ?

File: javers-core/src/test/groovy/org/javers/core/json/typeadapter/commit/CommitPropertiesConverterTest.groovy [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+                .map { element -> element['key'] }
+                .map { primitive -> primitive.getAsString() }
+                .collect(Collectors.toList())
+
+        assert 'name' in keys
```
Comment (bartoszwalacik) [inline javers-core/src/test/groovy/org/javers/core/json/typeadapter/commit/CommitPropertiesConverterTest.groovy]: more important is `!keys.contains('age')`
(`assert` keyword is optional here)

File: javers-core/src/test/groovy/org/javers/core/json/typeadapter/commit/CommitPropertiesConverterTest.groovy (hunk 1 above)
Comment (tau3) [reply javers-core/src/test/groovy/org/javers/core/json/typeadapter/commit/CommitPropertiesConverterTest.groovy]: yep, that is definitely more expressive :)

**Comment by bartoszwalacik** [comment]
Sergey Rozhnov your commits are  not linked with your github account, you can fix it by registering the email from these commits in your github account. Then you will be listed as a JaVers contributor

**Pull request merged**